examples/LiftSim_baseline/DQN/README.md

```diff
@@ -6,9 +6,9 @@
 
 ## 依赖库
 
-- paddlepaddle >= 1.5.1
-- parl >= 1.1.2
-- rlschool >= 0.0.1
+- paddlepaddle==1.5.1
+- parl==1.1.2
+- rlschool>=0.0.1
```

Comment:
Please add a hyperlink for the rlschool repository here.

Reply:
done